bitbots_vision_tools/scripts/colorpicker.py

```diff
@@ -0,0 +1,230 @@
+#!/usr/bin/env python3
+
+import cv2
+import numpy as np
+import time
+import pickle
+import os
+import rospy
+import VisionExtensions
+from cv_bridge import CvBridge
+from sensor_msgs.msg import Image
+
+rospy.init_node('bitbots_colorpicker')
+rospy.loginfo('Initializing colorpicker...', logger_name="colorpicker")
+_cv_bridge = CvBridge()
+
+
+def click_and_crop(event, x, y, flags, param):
+	"""
+	Callback for the OpenCV cursor.
+
+	:param event: Event type
+	:param x: Mouse x position
+	:param y: Mouse y position
+	:param flags: Callback flags
+	:param param: Some unused parameter
```

Comment:
is this parameter necessary if is unused? 
If it is necessary would it make sense to initialize it with param=none, so calling the method is less confusing than having to call it with an unused parameter?

Reply:
This is a callback, so the parameters are required by OpenCV.

Reply:
The param itself if not documented in the OpenCV doku.